docs/toc.md

```diff
@@ -99,10 +100,13 @@ to know what's next -->
 ### [Getting started with .NET Core on Windows](core/tutorials/using-on-windows.md)
 ### [Getting started with .NET Core on macOS](core/tutorials/using-on-macos.md)
 ### [Getting started with .NET Core on Windows/Linux/macOS using the command line](core/tutorials/using-with-xplat-cli.md)
-### [Getting started with ASP.NET Core](core/tutorials/aspnet-core.md) 
+### [Writing Libraries with Cross Platform Tools](core/tutorials/libraries.md)
+### [Getting started with ASP.NET Core](core/tutorials/aspnet-core.md)
```

Comment:
Maybe change the link text here to "Developing ASP.NET Core applications" since we're splitting up the "getting started" series.